spotitude.py

```diff
@@ -39,3 +40,36 @@
 
         # Create visualization
         make_visualization(top_tracks_df)
+
+        eel.init(".")
+
+        @eel.expose
+        def create_playlist():
```

Comment:
Functionality works fine! Just need a docstring now!

Reply:
Alright, will add that. Do you prefer to have the setup and initialization of Eel and the playlist creation handling in a separate file (or maybe just a function)?

Reply:
A separate file would be nice, just so in the future when we add more functionality to the GUI, all the code relating to it is all self-contained.

On that note, we should also be specific on the function names since we're creating a pop-up for the visualization as well as a pop-up for the playlist. In this case, instead of `create_playlist()`, something along the lines of `create_playlist_button()`, `create_playlist_popup` or something similar.